### PR TITLE
whitespace, shebang, exec attribute fixes in /ms directory

### PR DESCRIPTION
**Pull request by vszakats**
* eliminate all tabs
* fix shebang
* set exec attributes for .pl files (in /ms)

### CODE REVIEW

**Comment by vszakats** [comment]
These were minor fixes, apparently too minor to mandate attention. Closing.

**Comment by richsalz** [comment]
Please don’t close things because you haven't heard from someone on the team.  Bandwidth is limited :)

**Comment by vszakats** [comment]
Kept the branch there just in case. Reopening and thanks for the feedback!

**Comment by richsalz** [comment]
@levitte , your thoughts?  i can merge if you approve.

**Comment by richsalz** [comment]
+1

File: ms/segrenam.pl [hunk 1]
```diff
@@ -7,17 +7,17 @@
 # first argument can specify custom suffix...
 $suffix=(@ARGV[0]=~/^\$/) ? shift(@ARGV) : "\$m";
 #################################################################
-# rename segments in COFF modules according to %map table below	#
-%map=(	".text" => "fipstx$suffix",				#
-	".text\$"=> "fipstx$suffix",				#
-	".rdata"=> "fipsrd$suffix",				#
-	".data" => "fipsda$suffix"	);			#
+# rename segments in COFF modules according to %map table below #
+%map=(  ".text" => "fipstx$suffix",                             #
```
Comment (levitte) [inline ms/segrenam.pl]: Nit pick: why two spaces after `(`?

Comment (levitte) [inline ms/segrenam.pl]: ... Ah!  Because that's what the tab expanded to, ok.  I'd reduce that to one space.

**Comment by levitte** [comment]
+1 pending space fix

**Comment by vszakats** [comment]
Space fix done. (and squashed)

**Comment by levitte** [comment]
+1 confirmed.  @richsalz, ready to merge?

**Comment by richsalz** [comment]
yup :)

**Pull request closed without merging**